File: datastruct/set/Set.java
public interface Set<E> {

    //  向集合添加元素e
    void add(E e);

    //  判断集合中是否存在元素e
    boolean contains(E e);

    //  获取集合大小
    int getSize();

    //  判断集合是否为空
    boolean isEmpty();

    //  从集合中删除元素e
    void remove(E e);
}

File: datastruct/set/set.py
from abc import ABCMeta, abstractmethod

class Set(metaclass = ABCMeta):

    @abstractmethod
    def add(self, ele):
        pass

    @abstractmethod
    def remove(self, ele):
        pass

    @abstractmethod
    def contains(self, ele):
        pass

    @abstractmethod
    def get_size(self):
        pass

    @abstractmethod
    def is_empty(self):
        pass
